release-notes/4.7.17.md

```diff
@@ -0,0 +1,226 @@
+
+- **[CRM-19657](https://issues.civicrm.org/jira/browse/CRM-19657) recurring contribution receipt comes from logged in user ([9856](https://github.com/civicrm/civicrm-core/pull/9856) and [9842](https://github.com/civicrm/civicrm-core/pull/9842))**
+
+- **[CRM-19960](https://issues.civicrm.org/jira/browse/CRM-19960) Drupal 8 Standard Page title does not immediately disappear on contact record screen ([9770](https://github.com/civicrm/civicrm-core/pull/9770))**
+
+- **Remove mispelt param. ([9853](https://github.com/civicrm/civicrm-core/pull/9853))**
+
+- **[CRM-19972](https://issues.civicrm.org/jira/browse/CRM-19972) Add in Post upgrade message for addition of manage tags permission ([9808](https://github.com/civicrm/civicrm-core/pull/9808))**
+
+- **[CRM-19184](https://issues.civicrm.org/jira/browse/CRM-19184) Activities target contact list does not truncate  ([9846](https://github.com/civicrm/civicrm-core/pull/9846))**
+
+- **[CRM-20022](https://issues.civicrm.org/jira/browse/CRM-20022) Add Batch Title to Columns, Grouping tabs on Contribution Summary report ([9838](https://github.com/civicrm/civicrm-core/pull/9838))**
+
+- **[CRM-18811](https://issues.civicrm.org/jira/browse/CRM-18811) MySQL table and column aliases can contain spaces ([8548](https://github.com/civicrm/civicrm-core/pull/8548))**
+
+- **[CRM-20015](https://issues.civicrm.org/jira/browse/CRM-20015) token values with ampersand cause error when generating docx letter ([9834](https://github.com/civicrm/civicrm-core/pull/9834))**
+
+- **[CRM-19964](https://issues.civicrm.org/jira/browse/CRM-19964) Event registration creates contribution without line item ([9774](https://github.com/civicrm/civicrm-core/pull/9774))**
+
+- **[CRM-19950](https://issues.civicrm.org/jira/browse/CRM-19950) Wrong param passed to checkPermission function ([9832](https://github.com/civicrm/civicrm-core/pull/9832) and [9810](https://github.com/civicrm/civicrm-core/pull/9810))**
+
+- **[CRM-18464](https://issues.civicrm.org/jira/browse/CRM-18464) Database Error upgrading to Civicrm 4.7 ([9679](https://github.com/civicrm/civicrm-core/pull/9679))**
+
+- **[CRM-19979](https://issues.civicrm.org/jira/browse/CRM-19979) GroupContact.delete does not work for contacts with status of Pending ([9797](https://github.com/civicrm/civicrm-core/pull/9797))**
+
+- **I have added myself a second time - under WMF ([9805](https://github.com/civicrm/civicrm-core/pull/9805))**
+
+- **Release notes 4.7.16 (rest of 'em) ([9803](https://github.com/civicrm/civicrm-core/pull/9803))**
+
+- **remove markdown files in /docs ([9771](https://github.com/civicrm/civicrm-core/pull/9771))**
+
+- **civicrm_webtest.install - Restore access to edit message templates and tags ([9785](https://github.com/civicrm/civicrm-core/pull/9785))**
+
+- **Update Test generated examples ([9798](https://github.com/civicrm/civicrm-core/pull/9798))**
+
+- **Release notes 4.7.16 [wip] ([9781](https://github.com/civicrm/civicrm-core/pull/9781))**
```

Comment:
:smirk: yep. This is raw from the script I use to scrape the commits since 4.7.16.